opal/threads/threads.h

```diff
@@ -24,118 +24,6 @@
-        }                                                \
-        while (*(act)) {                                 \
-            opal_condition_wait((cnd), (lck));           \
-        }                                                \
-        if (opal_debug_threads) {                        \
-            opal_output(0, "Thread obtained %s:%d",      \
-                        __FILE__, __LINE__);             \
-        }                                                \
-        *(act) = true;                                   \
-    } while(0);
-#else
-#define OPAL_ACQUIRE_THREAD(lck, cnd, act)               \
-    do {                                                 \
-        OPAL_THREAD_LOCK((lck));                         \
-        while (*(act)) {                                 \
-            opal_condition_wait((cnd), (lck));           \
-        }                                                \
-        *(act) = true;                                   \
-    } while(0);
-#endif
-
-
-#if OPAL_ENABLE_DEBUG
-#define OPAL_RELEASE_THREAD(lck, cnd, act)              \
-    do {                                                \
-        if (opal_debug_threads) {                       \
-            opal_output(0, "Releasing thread %s:%d",    \
-                        __FILE__, __LINE__);            \
-        }                                               \
-        *(act) = false;                                 \
-        opal_condition_broadcast((cnd));                \
-        OPAL_THREAD_UNLOCK((lck));                      \
-    } while(0);
-#else
-#define OPAL_RELEASE_THREAD(lck, cnd, act)              \
-    do {                                                \
-        *(act) = false;                                 \
-        opal_condition_broadcast((cnd));                \
-        OPAL_THREAD_UNLOCK((lck));                      \
-    } while(0);
-#endif
-
-
-#define OPAL_WAKEUP_THREAD(cnd, act)        \
-    do {                                    \
-        *(act) = false;                     \
-        opal_condition_broadcast((cnd));    \
-    } while(0);
-
-/* provide a macro for forward-proofing the shifting
- * of objects between libevent threads - at some point, we
- * may revamp that threading model */
-
-/* post an object to another thread - for now, we
- * only have a memory barrier */
-#define OPAL_POST_OBJECT(o)     opal_atomic_wmb()
-
-/* acquire an object from another thread - for now,
- * we only have a memory barrier */
-#define OPAL_ACQUIRE_OBJECT(o)  opal_atomic_rmb()
-
-
-
-OPAL_DECLSPEC int  opal_thread_start(opal_thread_t *);
-OPAL_DECLSPEC int  opal_thread_join(opal_thread_t *, void **thread_return);
-OPAL_DECLSPEC bool opal_thread_self_compare(opal_thread_t*);
-OPAL_DECLSPEC opal_thread_t *opal_thread_get_self(void);
-OPAL_DECLSPEC void opal_thread_kill(opal_thread_t *, int sig);
-OPAL_DECLSPEC void opal_thread_set_main(void);
-
-END_C_DECLS
+#include "opal/mca/threads/threads.h"
```

Comment:
Ditto with `opal/threads/condition.h ` -- it looks like this file should be removed.